nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/pom.xml

```diff
@@ -80,6 +85,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-storage</artifactId>
+            <version>1.25.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-bigquery</artifactId>
+            <version>1.25.0</version>
```

Comment:
@danieljimenez Please rebase it and make the necessary changes (on dependency) that @pvillard31 suggested.